modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy

```diff
@@ -209,20 +215,20 @@ class CondaCache {
      * @return the conda environment prefix {@link Path}
      */
     @PackageScope
-    Path createLocalCondaEnv(String condaEnv) {
+    Path createLocalCondaEnv(String condaEnv, String binaryName = "conda") {
```

Comment:
Can we keep `binaryName` as a class attribute instead of passing as argument? or both `conda` and `mamba`can be used in the same class instance?

Reply:
> Can we keep binaryName as a class attribute instead of passing as argument?

Hmm, do you think that it's better to do use the following at class level

```groovy
        private useMamba = false
        private binaryName = useMamba ? "mamba" : "conda"
```

and then simply use the `binaryName` within the methods instead of argument?


> or both conda and mamba can be used in the same class instance?

I am not sure how to interpret that, but from what I understand, it's enough to rely on mamba (i.e. a single binary) CLI since it'll automatically locate `conda` if needed, without accomodating this in the Groovy code.

Reply:
> Hmm, do you think that it's better to do use the following at class level

Let's have a `binaryName` default to `conda` and switch to `mamba` when the flag is enabled

Reply:
Okay, I have accomodated the change request :)